test/e2e/scenarios/multiple_clusters.go

```diff
@@ -0,0 +1,14 @@
+package scenarios
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = PDescribe("Namespace selector based Network Policies for ingress between clusters, selecting multiple remote clusters", func() {
+	PIt("Should allow communication between pods in selected namespace across multiple clusters", func() {
+		By("creating pod1 with lable1 in cluster1 in namespace1")
```

Comment:
nit: looks like copy/paste issue. Please fix "label" throughout the file.

Reply:
Done